openml/extensions/sklearn/flow_functions.py

```diff
@@ -875,17 +895,59 @@ def _serialize_cross_validator(o):
 
     ret['oml-python:serialized_object'] = 'cv_object'
     name = o.__module__ + "." + o.__class__.__name__
-    value = OrderedDict([['name', name], ['parameters', parameters]])
+    value = OrderedDict([('name', name), ('parameters', parameters)])
     ret['value'] = value
 
     return ret
 
 
-def _check_n_jobs(model):
-    """
-    Returns True if the parameter settings of model are chosen s.t. the model
-    will run on a single core (if so, openml-python can measure runtimes)
+def _deserialize_cross_validator(value: 'OrderedDict[str, Any]', recursion_depth: int) -> Any:
+    model_name = value['name']
+    parameters = value['parameters']
+
+    module_name = model_name.rsplit('.', 1)
+    model_class = getattr(importlib.import_module(module_name[0]),
+                          module_name[1])
+    for parameter in parameters:
+        parameters[parameter] = flow_to_sklearn(
+            parameters[parameter], recursion_depth=recursion_depth + 1
+        )
+    return model_class(**parameters)
+
+
+def _format_external_version(model_package_name: str, model_package_version_number: str) -> str:
+    return '%s==%s' % (model_package_name, model_package_version_number)
```

Comment:
future pr: not json parsable. (issue)

Reply:
I chose this formatting to be in line with python's version formatting, but I agree that it becomes hard to maintain and impractical.